theme/templates/resource-landing-page/title-section.html

```diff
@@ -51,6 +51,12 @@
                         <br>
                         <span>Content files are required to publish the resource or make the resource public or discoverable.</span>
                 {% endif %}
+                {%  if cm.resource_type|lower == "collectionresource" %}
+                    {% if not cm.has_resources%}
+                        <br>
+                        <span>Collection should contain at least one resource to be made it published, public or discoverable.</span>
+                    {% endif %}
+                {% endif %}
```

Comment:
@hyi @pkdash I think if we change the message in hs_core to "Metadata Status: all required metadata provided.", there will be no message to tell user the resource is "Sufficient to make public or discoverable". But I think msg "Sufficient to make public or discoverable" is important.
I just had quick look at update_metadata_element() and add_metadata_element() in hs_core/views/__init__.py. They both only check "res.metadata.has_all_required_elements()" and then tell frontend whether it is sufficient or insufficient to make public or discoverable. I am guessing this check should be replaced by "res.can_be_public_or_discoverable". Then it should be able to fit collection res as well as other res types. But I am not sure....

Since this issue is affecting serial res types, it is better to open a new ticket to track and fix it. If you think it is ok, i would defer to @pkdash or @hyi .

For the current implementation of collection,  I will keep this part as is for now. Any suggestions?